DocumentVectorPipelineFunctions/BlobTriggerFunction.cs

```diff
@@ -42,37 +44,49 @@ public async Task Run([BlobTrigger("documents/{name}", Connection = "AzureBlobSt
     private async Task HandleBlobCreateEventAsync(BlobClient blobClient)
     {
         this._logger.LogInformation("Analyzing document using DocumentAnalyzerService from blobUri: '{blobUri}' using layout: {layout}", blobClient.Name, "prebuilt-read");
-        var operation = await documentAnalysisClient.AnalyzeDocumentFromUriAsync(
-            WaitUntil.Completed,
-            "prebuilt-read",
-            blobClient.Uri);
-
-        var result = operation.Value;
-
-        this._logger.LogInformation("Extracted content from '{name}', # pages {pageCount}", blobClient.Name, result.Pages.Count);
 
         var cosmosDBClientWrapper = await CosmosDBClientWrapper.CreateInstance(cosmosClient, this._logger);
 
-        int totalChunksCount = 0;
-        var batchChunkTexts = new List<TextChunk>();
-        foreach (var chunk in TextChunker.FixedSizeChunking(result))
+        using (var blobStream = await blobClient.OpenReadAsync())
```

Comment:
```suggestion
        using var blobStream = await blobClient.OpenReadAsync();
```

In recent C# you can avoid the extra level of indent by using a using statement without the block.